apps/public-docsite-v9/src/Concepts/Upgrade/FromV0/Components/Label.stories.mdx

```diff
@@ -0,0 +1,58 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Concepts/Upgrading/from v0/Components/Label Upgrade" />
+
+# Label Upgrade
+
+Fluent UI Northstar (v0) provides the `Label` control to allow users to classify content. Fluent UI v9 also provides a `Label`, but has a simpler API.
```

Comment:
```suggestion
Fluent UI Northstar (v0) provides the `Label` control to allow users to classify content. Fluent UI v9 also provides a `Label`, but has a different API.
```

Nit pick for sure but I would make an objective statement here rather than a subjective one about the relatively complexity of the APIs.